assignments/arrays.js

```diff
@@ -74,16 +74,32 @@ let inventory = [
 // [12, 14]
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+// The dealer can't recall the information for a car with an id of 33 on his lot. 
+// Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
+for (let i = 0; i < inventory.length; i++) {
+  if (i===33){
+    console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`);
+  } else {
+  console.log("this is not car 33!")  
+}
+};
+
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// The dealer needs the information on the last car in their inventory.  
+// What is the make and model of the last car in the inventory?  
+// Log the make and model into the console.
+
+for (let i = 0; i < inventory.length; i++) {
+if (i===50){
+  console.log(`${invetory[i].car_make}  ${inventory[i].car_year}`)
+} else {
+  break;
+}};
```

Comment:
Ayomide, how would you know which is the last entry in the array if the length of it changed constantly? For example, it gets updated every 30s. Would you go back to your code and edit `i === 50` every time to represent the current length of it? Of course you wouldn't!

The length property returns number of elements inside an array, therefore you already know how many entries is there are in total. Since arrays start indexing from 0, all you have to do to find it's last entry is to a perform a simple math operation on it's length, like so:

`inventory[inventory.length - 1];`

Conditional loop is not needed in this case.